src/tds/bulk.c

```diff
@@ -760,16 +761,28 @@ tds7_bcp_send_colmetadata(TDSSOCKET *tds, TDSBCPINFO *bcpinfo)
 		 * different from BCP format
 		 */
 		if (is_blob_type(bcpcol->on_server.column_type)) {
-			/* FIXME support multibyte string */
-			len = tds_dstr_len(&bcpinfo->tablename);
-			TDS_PUT_SMALLINT(tds, len);
-			tds_put_string(tds, tds_dstr_cstr(&bcpinfo->tablename), len);
+			converted_name = tds_convert_string(tds, tds->conn->char_convs[client2ucs2], tds_dstr_cstr(&bcpinfo->tablename), (int)tds_dstr_len(&bcpinfo->tablename), &converted_len);
+			if (!converted_name) {
+				return TDS_FAIL;
+			}
+
+			/* UTF-16 length is always size / 2 even for 4 byte letters (yes, 1 letter of length 2) */
+			TDS_PUT_SMALLINT(tds, converted_len / 2);
```

Comment:
Hire `TDS_PUT_SMALLINT` is used, but for column name length `tds_put_byte`.
Is this intentional?